src/cloud-api-adaptor/pkg/adaptor/cloud/cloud.go

```diff
@@ -74,8 +77,18 @@ func (s *cloudService) removeSandbox(id sandboxID) error {
 }
 
 func NewService(provider provider.Provider, proxyFactory proxy.Factory, workerNode podnetwork.WorkerNode,
-	podsDir, daemonPort, aaKBCParams string) Service {
+	secureComms bool, secureCommsInbounds, secureCommsOutbounds, kbsAddress, podsDir, daemonPort, aaKBCParams, sshport string) Service {
 	var err error
+	var sshClient *wnssh.SshClient
+
+	if secureComms {
```

Comment:
Same comment as previously for the strings. Better to put it as constants for easier readability

Reply:
I think it is more confusing as constants - see comment in forwarder